src/components/CharacterLimitMainText/CharacterLimit.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react';
+
+import CharacterLimit from './CharacterLimit.tsx';
+
+describe('CharacterLimit Component', () => {
+  it('render the component without svg', () => {
+    render(
+      <CharacterLimit id={'test'} maxLength={10} svg={null} value={'1234'} />
+    );
+    const characterLimit = screen.getByLabelText('characterLimit');
+    expect(characterLimit).toBeInTheDocument();
+  });
+
+  it('render a svg component', () => {
+    render(
+      <CharacterLimit id={'test'} maxLength={10} svg={<svg />} value={'1234'} />
+    );
+    const characterLimit = screen.getByLabelText('characterLimitWithIcon');
+    expect(characterLimit).toBeInTheDocument();
```

Comment:
Uma coisa que voce pode fazer é já pegar o getByLabelText diretamente do render, fazendo ficar assim.

```suggestion
    const { getByLabelText } = render(
      <CharacterLimit id={'test'} maxLength={10} svg={<svg />} value={'1234'} />
    );
    const characterLimit = getByLabelText('characterLimitWithIcon');
    expect(characterLimit).toBeInTheDocument();
```

Reply:
Isso é desencorajado pelo pessoal da lib man!

Existem casos que o `getBy...` não pega o estado atual e ai a recomendação é usar o `screen`!

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen